addons/payment_stripe/controllers/main.py

```diff
@@ -113,7 +117,9 @@ def stripe_webhook(self):
                         )
                         self._include_setup_intent_in_feedback_data(setup_intent, data)
                     # Handle the feedback data crafted with Stripe API objects as a regular feedback
-                    request.env['payment.transaction'].sudo()._handle_feedback_data('stripe', data)
+                    request.env['payment.transaction'].with_user(
+                        SUPERUSER_ID
+                    ).sudo()._handle_feedback_data('stripe', data)
```

Comment:
why is this change necessary?
If it's for what I believe it is, shouldn't it be modified in the core payment code ? :thinking:

Reply:
Without the superuser_id, the log notes from the webhook are displayed as "Public User" instead of "OdooBot". I don't know if this should be modified in the core payment code but if you think it should, I'll do it.

Reply:
Might be a good separate commit to make sure message/log notes posted during transactions processing (from webhook or returns from redirects) are posted as 'Odoobot'.

Doesn't make sense for me to do that kind of change only in the custom stripe code.
Also, `with_user(superuser_id)` has multiple side effects, including being always sudoed, which might not be something we want to apply for the whole `_handle_feedback_data` call.  I really prefer if the user change is only applied in the targeted places where messages/log notes are posted.